cps/kni.h

```diff
@@ -19,8 +19,11 @@
 #ifndef _GATEKEEPER_CPS_KNI_H_
 #define _GATEKEEPER_CPS_KNI_H_
 
+#include "gatekeeper_net.h"
+
 int kni_change_mtu(uint8_t port_id, unsigned new_mtu);
 int kni_change_if(uint8_t port_id, uint8_t if_up);
+int kni_config(struct rte_kni *kni, struct gatekeeper_if *iface);
```

Comment:
The order of the includes in the source files allowed it to compile, but I agree that the include for rte_kni.h more naturally fits in cps/kni.h, and then we don't need to include it in multiple source files. Changed.